index.html

```diff
@@ -81,9 +81,9 @@
 					<div class="container">
 						<div class="col-md-12 col-md-offset-0">
 							<div class="animate-box">
-								<h2><code>jiant</code></h2>
-								<p>A software toolkit for research on general-purpose text understanding models.</p>
-								<p><a href="https://github.com/nyu-mll/jiant/blob/master/tutorials/setup_tutorial.md" class="btn btn-primary btn-lg btn-custom">Get Started</a> <a href="https://github.com/nyu-mll/jiant/" class="btn btn-primary btn-lg btn-custom">View on GitHub</a></p>
+								<h2 style="font-family: Menlo, Monaco, Consolas, Courier New, monospace;">jiant</h2></span>
+								<p>A software toolkit for research <br> on general-purpose text understanding models.</p>
+								<p><a href="https://github.com/nyu-mll/jiant" class="btn btn-primary btn-lg btn-custom">jiant v1.3 (Legacy)</a> <a href="https://github.com/jiant-dev/jiant" class="btn btn-primary btn-lg btn-custom">jiant v2 (Pre-Alpha)</a></p>
```

Comment:
'Pre-Alpha' seems a bit menacing. I was assuming that we'd have something usable (if limited in scope), that we would actually suggest people use as the default. In that case, 'beta'?